tests/build_/test_build_request.py

```diff
@@ -1652,7 +1650,7 @@ def test_prod_is_custom_base_image(self, tmpdir, base_image, is_custom):
         kwargs = get_sample_prod_params()
         kwargs['base_image'] = base_image
         build_request.set_params(**kwargs)
-        build_json = build_request.render()
+        build_json = build_request.render()  # noqa
```

Comment:
That's also a solution, but that was a functional change beyond my remit.

I can make that change though.